vackup

```diff
@@ -123,7 +177,7 @@ cmd_export() {
       -v "$VOLUME_NAME":/volume-data \
       -v "$DIRECTORY":/mount-volume \
       busybox \
-      tar -cvzf /mount-volume/"$FILE_NAME" -C /volume-data .;
+      /bin/ash -c 'tar -cvzf /mount-volume/'"'$FILE_NAME'"' -C /volume-data . && chown '"'${OWNER:-"$(id -u):$(id -g)"}'"' /mount-volume/'"'$FILE_NAME'";
```

Comment:
I'd recommend `/bin/sh` here over `ash`, which makes the command more portable across images.  Ash is not in every official image, but `sh` is, and on alpine-based images, `sh` calls `ash` anyway.

Reply:
AFAIK, "*The default shell inside BusyBox is ash, also known as the Almquist shell or sh*", so since we are using the busybox image ash should always be there, but it doesn't really matter, so roger that